packages/apps/web-viewer-test/package.json

```diff
@@ -58,9 +62,9 @@
     "not op_mini all"
   ],
   "scripts": {
-    "start": "react-scripts --max_old_space_size=4096 start",
+    "start": "vite",
     "build": "",
-    "build:frontend": "react-scripts --max_old_space_size=4096 build",
+    "build:frontend": "vite build",
```

Comment:
why do we have both a build and build:frontend?
i think previously we did it because build times were too long, but should be much faster with vite